pkgs/applications/window-managers/hyprwm/hyprland-plugins/hyprland-plugins.nix

```diff
@@ -0,0 +1,49 @@
+let
+  # shared src for upstream hyprland-plugins repo
+  # function generating derivations for all plugins in hyprland-plugins
+  hyprland-plugins =
+    builtins.mapAttrs
+      (
+        name: description:
+        (
+          {
+            lib,
+            cmake,
+            fetchFromGitHub,
+            hyprland,
+            mkHyprlandPlugin,
+          }:
+          let
+            hyprland-plugins-src = fetchFromGitHub {
+              owner = "hyprwm";
+              repo = "hyprland-plugins";
+              rev = "d92d9d6cd93b731143119cbcab54dcc7a0481131";
+              hash = "sha256-Bw3JRBUZg2kmDwxa/UHvD//gGcNjbftTj2MSeLvx1q8=";
+            };
+          in
+          mkHyprlandPlugin hyprland {
+            pluginName = name;
+            inherit (hyprland) version;
```

Comment:
So this inherits Hyprland version, if Hyprland is updated to 0.42 for example, it would rebuild the package and "label" the plugins as 0.42, even without actually updating the hyprland-plugins?

Instead of this, properly tagging upstream plugin releases would be nice. If that can not be arranged, it should use the unstable versioning scheme.

Reply:
Yup, you're right. I just made the v0.41.1 tag and changed the code here to point to it. Version is also a separate key now.

Reply:
Sweet, would love an update script, but let's just merge it for now.